src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -259,31 +248,27 @@ InferenceEngine::Parameter ExecNetwork::GetMetricLegacy(const std::string &name,
         auto option = engConfig._config.find(CONFIG_KEY(CPU_THROUGHPUT_STREAMS));
         IE_ASSERT(option != engConfig._config.end());
         auto streams = std::stoi(option->second);
-        IE_SET_METRIC_RETURN(OPTIMAL_NUMBER_OF_INFER_REQUESTS, static_cast<unsigned int>(
-            streams ? streams : 1));
+        IE_SET_METRIC_RETURN(OPTIMAL_NUMBER_OF_INFER_REQUESTS, static_cast<unsigned int>(streams ? streams : 1));
     } else {
-        IE_THROW() << "Unsupported ExecutableNetwork metric: " << name;
+        OPENVINO_THROW("Unsupported property: ", name);
     }
+    OPENVINO_SUPPRESS_DEPRECATED_END
 }
 
-InferenceEngine::Parameter ExecNetwork::GetMetric(const std::string &name) const {
-    if (_graphs.empty())
-        IE_THROW() << "No graph was found";
+ov::Any CompiledModel::get_metric(const std::string& name) const {
```

Comment:
Why do you need this method?! New API doesn't have metrics. Let's keep only `get_property()`.

Reply:
ok, move get_metric to get_property